tests/data/worldstrat/data.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import hashlib
+import os
+import shutil
+import tarfile
+from datetime import datetime, timedelta
+
+import numpy as np
+import pandas as pd
+import rasterio
+from PIL import Image
+
+
+def create_dummy_worldstrat(root: str, img_size: int = 64) -> None:
+    """Create dummy WorldStrat dataset."""
+    os.makedirs(root, exist_ok=True)
+
+    tiles = {'train': ['AOI001', 'AOI002'], 'val': ['AOI003'], 'test': ['AOI004']}
+
+    metadata = []
+    split_info = []
+
+    # Generate 4 dates for time series
+    base_date = datetime(2021, 1, 1)
+    dates = [base_date + timedelta(days=i * 30) for i in range(4)]
+
+    for split, tile_list in tiles.items():
+        for tile in tile_list:
+            if os.path.exists(os.path.join(root, tile)):
+                shutil.rmtree(os.path.join(root, tile))
+            tile_dir = os.path.join(root, tile)
+            l1c_dir = os.path.join(tile_dir, 'L1C')
+            l2a_dir = os.path.join(tile_dir, 'L2A')
+            os.makedirs(l1c_dir, exist_ok=True)
+            os.makedirs(l2a_dir, exist_ok=True)
+
+            # High-res images (single timestep)
+            hr_ps = np.random.randint(0, 255, (4, img_size, img_size), dtype=np.uint16)
+            with rasterio.open(
```

Comment:
Lot of warnings in the tests:
```
tests/datasets/test_worldstrat.py: 6 warnings
  /opt/hostedtoolcache/Python/3.13.1/x64/lib/python3.13/site-packages/rasterio/__init__.py:356: NotGeoreferencedWarning: Dataset has no geotransform, gcps, or rpcs. The identity matrix will be returned.
    dataset = DatasetReader(path, driver=driver, sharing=sharing, **kwargs)
```
If the original dataset has geospatial metadata, then add it to the files. If not, then use PIL to read/write the files.